wayland-logout.1

```diff
@@ -0,0 +1,41 @@
+.Dd $Mdocdate: November 27 2020 $
+.Dt WAYLAND-LOGOUT 1
+.Os
+.Sh NAME
+.Nm wayland-logout
+.Nd generic logout command for wayland compositors
+.Sh DESCRIPTION
+.Nm
+is a compositor agnostic command for simulating a
+logout with any wayland complaint compositor.
+.Pp
+It sends a
+.Sy SIGINT
+.Xr signal 3
+to the running compositor that is obtained by looking
+at the
+.Sx environment
+variables.
+.Sh ENVIRONMENT
+The command uses the
+.Ev WAYLAND_DISPLAY
+and
+.Ev XDG_RUNTIME_DIR
+.Nm
+environment variables to get the
+.Xr socket 2
+corresponding to the compositor instance.
```

Comment:
```suggestion
corresponding to the compositor instance. .Sy SIGINT is sent to the pid owning the socket.
```

Reply:
This is the environment variables section.
We already referred to the `SIGINT` in the previous section.
Seems a bit superfluous.